content/references/protocol-reference/transactions/transaction-types/nftokenburn.md

```diff
@@ -123,15 +123,15 @@ The `SetAccount` transaction allows the `MintAccount` field to be set or cleared
 ### MintedTokens
 
 
-The `MintedTokens` field is used to form the `TokenID` of a new object, to ensure the uniqueness of `NFToken` objects. If this field is not present, the value is 0.
+The `MintedTokens` field is used to form the `NFTokenID` of a new object, to ensure the uniqueness of `NFToken` objects. If this field is not present, the value is 0.
```

Comment:
I believe the `MintedTokens` field becomes `MintedNFTokens`.

Also, earlier in the document, there's mention of a `MintAccount` field.  I think that field was actually called `Minter` by the previous implementation.  That field is now called `NFTokenMinter`.